show/bgp_common.py

```diff
@@ -353,25 +354,29 @@ def show_routes(args, namespace, display, verbose, ipver):
     else:
         back_end_intf_set = None
     # get all the other arguments except json that needs to be the last argument of the cmd if present
+    # Handling of multi-asic show ip/v6 route summary will print directly FRR result of each namespace
+    check_summary_set = set(["sum", "summ", "summa", "summar", "summary"])
     for arg in args:
         arg_strg += str(arg) + " "
         if str(arg) == "json":
             found_json = 1
         elif str(arg) == "tables":
             found_tables = 1
+        elif set([str(arg)]).issubset(check_summary_set):
```

Comment:
Not sure I understand the logic. Why not:
```
if str(arg) in check_summary_set:
```

Reply:
The logic is to allow partial matching so that user need not specify the complete word "summary".
This logic is needed so that if user specify something like "umm" it will not be accepted.  it needs to start with "sum" and maybe additional matching chars after "sum".